src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -247,3 +256,28 @@ public String getHelpText() {
     return getInfoText();
   }
 }
+
+
+class AdvancedRedirectStrategy extends LaxRedirectStrategy {
```

Comment:
@RoiEXLab Just noticed this after pulling your PR.  This is a violation of the Checkstyle OneTopLevelClass rule.  It didn't trigger a failure because you fixed a VariableDeclarationUsageDistance violation on line 135 at the same time.  You can fix it by making `AdvancedRedirectStrategy` a nested private member of `PropertiesDiceRoller`:

```java
public class PropertiesDiceRoller implements IRemoteDiceServer {
  // . . .
  private static class AdvancedRedirectStrategy extends LaxRedirectStrategy {
    // . . .
  }
}
```

Not a big deal.  If you can fix it before this PR gets merged, great.  Otherwise, we'll hit it eventually.